glue/core/visual.py

```diff
@@ -107,7 +107,9 @@ def preferred_cmap(self):
     @preferred_cmap.setter
     def preferred_cmap(self, value):
         if isinstance(value, str):
-            self._preferred_cmap = get_cmap(value)
+            for i, element in enumerate(colormaps.members):
+                if element[0] == value:
+                    self._preferred_cmap = element[1]
```

Comment:
Well, if I do so I would get an error thrown since some native `glue` `cmaps` are not in the standard `matplotlib` library. Is there an alternative which would make this acceptable like a `try-except` statement?